docs/content/modules/tensor_mechanics/introduction/Stresses.md

```diff
@@ -95,15 +95,15 @@ The integration of the model is performed using a combination of Material and Di
 * `HEVPStrengthUOBase`
 
 
-##Creep and Plasticity with Recompute Materials
-In addition to the User Object based plasticity models, another set of plasticity and creep material models have been developed with the `DiscreteMaterial` class, which allows for iterations within the material itself.  These iterative materials are designated by `Recompute` at the start of the class name. These classes use a Radial Return von Mises, or J2, approach to determine the inelastic, creep, damage, or plasticity strain increments at each time step.  The advantage of the recompute materials is the ability to combine multiple inelastic stress calculations, such as creep and plasticity. The [Recompute Radial Return Mapping Algorithm](/Materials/RecomputeRadialReturn.md) discusses the general algorithm to return the stress state to the yield surface.
+##Creep and Plasticity with StressUpdate Materials
+In addition to the User Object based plasticity models, another set of plasticity and creep material models have been developed with the `DiscreteMaterial` class, which allows for iterations within the material itself.  These iterative materials are designated by `StressUpdate` at the end of the class name. These classes use a Radial Return von Mises, or J2, approach to determine the inelastic, creep, damage, or plasticity strain increments at each time step.  The advantage of the recompute materials is the ability to combine multiple inelastic stress calculations, such as creep and plasticity. The [Stress Update Radial Return Mapping Algorithm](/Materials/RadialReturnStressUpdate.md) discusses the general algorithm to return the stress state to the yield surface.
```

Comment:
`DiscreteMaterial` should be replaced with `StressUpdateBase`, and replace "recompute" with "stress update" in a few places.